Makefile

```diff
@@ -50,8 +50,8 @@ goyacc:
 	$(GOBUILD) -o bin/goyacc parser/goyacc/main.go
 
 parser: goyacc
-	bin/goyacc -o /dev/null -xegen $(TEMP_FILE) parser/parser.y
-	bin/goyacc -o parser/parser.go -xe $(TEMP_FILE) parser/parser.y 2>&1 | egrep "(shift|reduce)/reduce" | awk '{print} END {if (NR > 0) {print "Find conflict in parser.y. Please check y.output for more information."; system("rm -f $(TEMP_FILE)"); exit 1;}}'
+	bin/goyacc -o /dev/null parser/parser.y
+	bin/goyacc -o parser/parser.go parser/parser.y 2>&1 | egrep "(shift|reduce)/reduce" | awk '{print} END {if (NR > 0) {print "Find conflict in parser.y. Please check y.output for more information."; system("rm -f $(TEMP_FILE)"); exit 1;}}'
 	rm -f $(TEMP_FILE)
```

Comment:
Do we still need the `TEMP_FILE`?

Reply:
If we do not need TEMP_FILE, we should remove other lines. Such as line 55.